ui/parcel.js

```diff
@@ -1,23 +1,31 @@
+/* eslint no-console: 0 */
```

Comment:
Most of the changes in this file are just code golfing.... sorry for the diff noise.